content/en/installing-pixie/03-install-guides/01-community-cloud-for-pixie.md

```diff
@@ -0,0 +1,120 @@
+---
+title: "Community Cloud for Pixie"
+metaTitle: "Install | Install Guides | Community Cloud for Pixie"
+metaDescription: "Getting started guide to setup Pixie"
+order: 1
+---
+
+Please review Pixie's [requirements](/installing-pixie/requirements) to make sure that your Kubernetes cluster is supported. 
+
+## 1. Sign up
+
+Visit our [product page](https://work.withpixie.ai/) and sign up with your google or gsuite account.
+
+## 2. Set up a Kubernetes environment (optional)
+
+If you don't have a Kubernetes cluster available, you can set up Minikube as a local sandbox environment. Instructions are available [here](/installing-pixie/setting-up-k8s/minikube-setup).
+
+## 3. Install the Pixie CLI
+
+You can install Pixie's CLI tool in one of 4 ways:
+
+### Using the install script (easiest)
+``` bash
+# Copy and run command to install the Pixie CLI.
+bash -c "$(curl -fsSL https://withpixie.ai/install.sh)"
+```
+
+### Directly downloading the binary
+
+``` bash
+# Download the latest Pixie binary.
+curl -o px https://storage.googleapis.com/pixie-prod-artifacts/cli/latest/cli_darwin_amd64
+
+# Check the signature matches.
+sha256sum px
+
+# Make it executable.
+chmod +x px
+
+# (Optional) Move it to another location that has executables.
+mv px /usr/local/bin
+```
+
+### Using Docker
+
+``` bash
+alias px="docker run -i --rm -v ${HOME}/.pixie:/root/.pixie pixielabs/px"
+```
+
+### Using Debian package
+
+- [https://storage.googleapis.com/pixie-prod-artifacts/cli/latest/pixie-px.x86_64.deb](https://storage.googleapis.com/pixie-prod-artifacts/cli/latest/pixie-px.x86_64.deb)
+- [https://storage.googleapis.com/pixie-prod-artifacts/cli/latest/pixie-px.x86_64.deb.sha256](https://storage.googleapis.com/pixie-prod-artifacts/cli/latest/pixie-px.x86_64.deb.sha256)
+
+``` bash
+# Install Pixie .deb package.
+dpkg -i pixie-px.x86_64.deb
+```
+
+### Using RPM
+
+- [https://storage.googleapis.com/pixie-prod-artifacts/cli/latest/pixie-px.x86_64.rpm](https://storage.googleapis.com/pixie-prod-artifacts/cli/latest/pixie-px.x86_64.rpm)
+- [https://storage.googleapis.com/pixie-prod-artifacts/cli/latest/pixie-px.x86_64.rpm.sha256](https://storage.googleapis.com/pixie-prod-artifacts/cli/latest/pixie-px.x86_64.rpm.sha256)
+
+``` bash
+# Install Pixie .rpm package.
+rpm -i pixie-px.x86_64.rpm
+```
+
+## 4. Deploy Pixie 🚀
+
+You can deploy Pixie to your Kubernetes cluster with the Pixie CLI, [YAML](/installing-pixie/install-schemes/yaml), or [Helm](/installing-pixie/install-schemes/helm).
```

Comment:
Rearrange this section:

Pixie's CLI is the fastest and easiest way to deploy Pixie:

``` bash
# List Pixie deploy options
px deploy --help

# Deploy the Pixie Platform in your K8s cluster by running:
px deploy
```

You can also deploy Pixie using [YAML](/installing-pixie/install-schemes/yaml) or [Helm](/installing-pixie/install-schemes/helm).